common/common.mak

```diff
@@ -13,7 +13,7 @@ ifeq ($(shell echo $(PARSER_URL) | grep '^[a-z][-+.0-9a-z]*://'),)
 endif
 endif
 
-TS ?= tree-sitter
+TS ?= npx tree-sitter-cli
```

Comment:
Nope.
```suggestion
TS ?= tree-sitter
```

Reply:
I have reverted these changes as you asked.